src/internal.c

```diff
@@ -19077,17 +19120,14 @@ int ProcessReplyEx(WOLFSSL* ssl, int allowSocketErr)
             #endif
                 if (ret < 0) {
                     WOLFSSL_MSG("VerifyMacEnc failed");
-                    WOLFSSL_ERROR(ret);
```

Comment:
this removes an error from the error queue with compat. builds, is it necessary to remove it?